### PR TITLE
Nonpr 2764

### PR DESCRIPTION
**Pull request by DanielConnelly**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -145,6 +146,7 @@ notableEvents = [
   {
     notableEvent = "vat-return-ui"
     displayName = "VAT return UI"
+    pluralDisplayName = "VAT return UIs"
```
Comment (greg-gajda) [inline conf/application.conf:149]: Shouldn't that be VAT returns UI ?

File: conf/application.conf [hunk 2]
```diff
@@ -225,6 +233,7 @@ notableEvents = [
   {
     notableEvent = "itsa-cgt-disposal-ppd"
     displayName = "ITSA CGT Report and Pay CGT on Property"
+    pluralDisplayName = "ITSA CGT Report and Pay CGT on Propertys"
```
Comment (greg-gajda) [inline conf/application.conf:236]: Properties not Propertys

Comment (greg-gajda) [reply conf/application.conf:236]: but isn't plural in this case affecting Reports ?

Comment (DanielConnelly) [reply conf/application.conf:236]: Discussed on call.

**Review by crashposition — APPROVED** [review]

**Pull request merged**